src/ring_theory/polynomial/homogeneous.lean

```diff
@@ -0,0 +1,285 @@
+/-
+Copyright (c) 2020 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin
+-/
+
+import data.mv_polynomial
+import data.fintype.card
+import tactic
+#print tactic.interactive.squeeze_simp
```

Comment:
```suggestion

```

Reply:
Why did this pass CI? @bryangingechen @robertylewis 
I remember putting this `#print` statement here exactly because I wanted CI to remind me that I should minimize imports... but apparently that trick doesn't work.

Reply:
CI doesn't check for a lack of `#print` statements. If you want a warning, `#check` a declaration that doesn't exist.

Reply:
Ooh, I thought it would complain about printing output...

Reply:
It should be possible to set it up to do that. I can't work on it right now, but please make an issue if you think it's a feature worth having and I'll try to come back to it when I can.